src/addons/transitions/ReactTransitionGroup.js

```diff
@@ -17,18 +17,28 @@ var ReactTransitionChildMapping = require('ReactTransitionChildMapping');
 var assign = require('Object.assign');
 var emptyFunction = require('emptyFunction');
 
+function _defaultRender(childrenToRender) {
```

Comment:
I don’t like that we are leaking `this` to the consumer. If consumer passes the props, they already have information about the props, so there is no need to pass them or the instance back.. I think.

Maybe, rather than make this a default prop, let’s make it optional and instead branch inside `render()` method? 

```js
var render = this.props.render || this._renderWrapper;
return render(children);
```